src/Documentation/Advanced-Concepts/Heterogeneous-Silos.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Heterogeneous silos
+---
+
+# Heterogeneous silos
+On a given cluster, silos can support a different set of grain types:
```

Comment:
Yes the file is used in the Heterogenous page?

I can add the visio file if you want, but it is not a very complicated diagram